packages/client-ts/src/client.ts

```diff
@@ -208,6 +210,39 @@ class Client implements IClient {
     return this.config
   }
 
+  public async getPublicProfile(did: string,
+    contextName: string,
+    profileName: string = "basicProfile",
+    fallbackContext: string | null = "Verida: Vault",
+    useCache: boolean = true): Promise<ProfileDocument> {
```

Comment:
As it's an optional parameter and the default is to use cache, the parameter should actually be `ignoreCache = false`.

Because, from a DX perspective, not defining a parameter usually means it's falsy: `ignoreCache = undefined` is equivalent to `ignoreCache = false` and `useCache = undefined` is equivalent to `useCache = false`. Because we want to use the cache by default, we can just flip the name and meaning of the parameter `userCache` to `ignoreCache`.

Also in that case, the data URL specify a `ignoreCache` param, so it's more straightforward to pass `ignoreCache=${ignoreCache}` rather than `ignoreCache=${!useCache}`.

Reply:
Agreed.